app/models/ext_management_system.rb

```diff
@@ -154,6 +154,7 @@ def self.create_from_params(params, endpoints, authentications)
   supports_not :cloud_subnet_create
   supports_not :cloud_tenant_mapping
   supports_not :cloud_volume_create
+  supports_not :cloud_volume_snapshots_create
```

Comment:
I think we also wouldn't need this one then?

Reply:
removed